RobotLib/src/main/java/com/acmerobotics/library/dashboard/config/Configuration.java

```diff
@@ -0,0 +1,61 @@
+package com.acmerobotics.library.dashboard.config;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by ryanbrott on 10/13/17.
+ */
+
+public class Configuration {
+    private Map<String, Option> options;
+
+    public Configuration() {
+        options = new HashMap<>();
+    }
+
+    public void addOptionsFromClass(Class<?> klass) {
```

Comment:
is there a distinction between klass and clazz?

Reply:
Nope 😄